doc/source/install/k8s-sysadmin.rst

```diff
@@ -39,58 +41,48 @@ Run multinode test
 
 On the workstation *myhost.domain.org*, get and prepare deployment scripts.
 
+Set up straightforward ssh access, using:
+  - optionally, an ssh client configuration file
+  - optionally, a parallel  configuration file (see example below)
+
+.. literalinclude:: ../../../admin/tools/docker/deployment/kubernetes/sshloginfile.example.in2p3
+   :language: bash
+   :linenos:
+
 .. code-block:: bash
 
    git clone git@github.com:lsst/qserv.git
-   cd ${SRC_DIR}/qserv/admin/tools/docker/deployment/kubernetes
-   # admin/ contains script which require ssh access to node
-   # orchestration/ is copied to k8s orchestration node and contains scripts
-   # which manage pods
-   # Create configuration file
-   cp env.example.sh env.sh
+   cd ${SRC_DIR}/qserv/admin/tools/docker/deployment/kubernetes/sysadmin
+   # Destroy kubernetes cluster if it exists
+   ./kube-destroy.sh
+   # Create kubernetes cluster
+   ./kube-create.sh
 
-In :file:`env.sh`, set your container configuration (qserv images, attached volumes, ...):
 
-.. literalinclude:: ../../../admin/tools/docker/deployment/kubernetes/env.example.sh
-   :language: bash
-   :linenos:
+On each machine running kubectl client, create a directory readable by all kubectl users,
+for example `/qserv/kubernetes`.
 
-You also need to create file below in `~/.lsst/qserv-cluster`:
+Then create files below inside it:
   - an environment file named `env-infrastructure.sh` with all node hostnames (see example below)
-  - optionally, an ssh client configuration file
 
-.. literalinclude:: ../../../admin/tools/docker/deployment/kubernetes/env.example.sh
+.. literalinclude:: ../../../admin/tools/docker/deployment/kubernetes/env-infrastructure.example.in2p3.sh
    :language: bash
    :linenos:
 
-Run Openstack provisioning script and check `~/.lsst/qserv-cluster` directory to see examples.
-
-Then, install Kubernetes, Qserv and launch multinodes integration tests.
-
-.. code-block:: bash
+- an environment file named `env.sh` with all kubernetes configuration (qserv images, attached volumes, ...)
```

Comment:
Is there an env.sh.example?

Reply:
Yes, in admin/tools/docker/deployment/kubernetes